FinSearchUnified/Tests/Bundle/SearchBundleFindologic/QueryBuilder/SearchQueryBuilderTest.php

```diff
@@ -380,4 +380,38 @@ public function testForceOriginalQueryIsNotTreatedAsFilter()
 
         $this->assertSame(1, $params['forceOriginalQuery']);
     }
+
+    public function testShopTypeParameter()
+    {
+        $searchNavigationRequest = new SearchRequest();
+        $queryBuilder = new SearchQueryBuilder(
+            $this->installerService,
+            $this->config,
+            null,
+            $searchNavigationRequest
+        );
+
+        $queryBuilder->addQuery('shopware5');
+
+        $params = $searchNavigationRequest->getParams();
+        $this->assertArrayHasKey('query', $params);
+        $this->assertSame('shopware5', $params['query']);
+    }
```

Comment:
This also applies to the other test:
- You are currently not testing the correct parameter, as you just set the query and test it
- As the parameter should be added by default, you don't need any additional call to e.g. `addQuery`
- It's usually a good idea to comment out your changes, and see if the test fails without it. This is not the case here

```suggestion
    public function testShopTypeParameterIsAddedByDefault()
    {
        $searchNavigationRequest = new SearchRequest();
        new SearchQueryBuilder(
            $this->installerService,
            $this->config,
            null,
            $searchNavigationRequest
        );

        $params = $searchNavigationRequest->getParams();
        $this->assertArrayHasKey('shopType', $params);
        $this->assertSame('Shopware5', $params['shopType']);
    }
```

Reply:
Thanks for the tip, we implemented the solution and now it seems ok.